solution/text-extractor/extractors/word.py

```diff
@@ -0,0 +1,21 @@
+import re
+
+import textract
+
+from .extractor import Extractor
+
+
+class WordExractor(Extractor):
+    file_types = (
+                  'docx',
+                  'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+                  'doc'
+                  )
+
+    def extract(self, file_path: str) -> str:
+        text = textract.process(file_path)
+        # cleans up weird characters
+        line = re.sub(r'(\n)+', ' ', text.decode('utf-8').encode('ascii', 'ignore').decode('utf-8'))
```

Comment:
Cleans up image characters and unicode stuff.  I used the retro doc and it had all the fun little images which we dont want in a word doc text search